provider/app.py

```diff
@@ -150,72 +39,21 @@ def testRoute():
 def healthRoute():
     return jsonify(generateHealthReport(consumers))
 
-
-def authorizedForTrigger(auth, consumer):
-    triggerURL = urlparse(consumer.triggerURL)
-
-    return (auth != None and consumer != None and triggerURL.username == auth.username and triggerURL.password == auth.password)
-
-
-def createAndRunConsumer(triggerFQN, params, record=True):
-    if app.config['TESTING'] == True:
-        logging.debug("Just testing")
-    else:
-        # generate a random uuid for new triggers
-        if not 'uuid' in params:
-            params['uuid'] = str(uuid.uuid4())
-
-        consumer = Consumer(triggerFQN, params)
-        consumer.start()
-        consumers.addConsumerForTrigger(triggerFQN, consumer)
-
-        if record:
-            database.recordTrigger(triggerFQN, params)
-
-
-def restoreTriggers():
-    for triggerDoc in database.triggers():
-        triggerFQN = triggerDoc['_id']
-        logging.debug('Restoring trigger {}'.format(triggerFQN))
-
-        try:
-            createAndRunConsumer(triggerFQN, triggerDoc, record=False)
-        except:
-            logging.warn('Skipping consumer due to caught exception: {}'.format(triggerDoc))
-
-
-def getMissingPostFields(fields):
-    missing = []
-    requiredFields = ['brokers', 'topic', 'triggerURL', 'isMessageHub']
-
-    if fields is None:
-        missing.extend(requiredFields)
-    else:
-        # Message Hub also requires 'username' and 'password fields'
-        if 'isMessageHub' in fields and fields['isMessageHub'] == True:
-            requiredFields.extend(['username', 'password'])
-
-        missing.extend([i for i in requiredFields if i not in fields])
-
-    if len(missing) > 0:
-        logging.info("Required fields are missing {}".format(missing))
-
-    return missing
-
-
 def main():
     logger = logging.getLogger()
     logger.setLevel(logging.INFO)
 
+    component = os.getenv('INSTANCE', 'messageHubTrigger-0')
```

Comment:
Exactly. Done for logging so you can tell which instance generated the log message.